README.md

```diff
@@ -24,6 +24,11 @@ First, install OpenAI via the [Composer](https://getcomposer.org/) package manag
 composer require openai-php/client
 ```
 
+If your project does not already include a PSR-18 client you have to install one:
```

Comment:
You don't need this step if you use php-http/discovery.

Reply:
Thank you @GromNaN 

I've updated the composer.json to enable auto installation.
Updated the README accordingly.

Do you agree how it is written now?